pkg/lib/patternfly/patternfly-5-overrides.scss

```diff
@@ -232,17 +233,17 @@ select.pf-v5-c-form-control {
   &__input {
     margin-block: auto;
     align-self: unset;
-    -moz-transform: unset;
+    transform: unset;
   }
 
   // Slightly shift the radio and check widgets
   &__input {
     // Shift up the checks/radios for (most) browsers
     transform: translateY(-1px);
     // Mozilla doesn't need the translation, so undo it
-    -moz-transform: none;
+    transform: none;
```

Comment:
It's this change which caused the checkbox pixel noise, and on second look that actually *was* supposed to be browser specific. I'll revert it and see how to quiesce stylelint.